client/src/components/FilterForm.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import { Search as SearchIcon } from 'lucide-react';
+
+type FilterProps = {
+  onSearchInput: (value: string) => void;
+};
+
+export default function FilterForm({ onSearchInput }: FilterProps) {
+  const [selectOption, setSelectOption] = useState({
+    location: '',
+    level: '',
+    salary: '',
+  });
+
+  return (
+    <div>
+      <form>
+        <div className="flex flex-col gap-3">
+          <div className="relative flex items-center">
+            <input
+              className="p-4 pl-10 rounded-lg border-[1.5px] w-full"
+              placeholder="Job Title or Company"
+              onChange={(e) => onSearchInput(e.currentTarget.value)}
+            />
+            <SearchIcon className="absolute left-3" color="#989898" />
+            <button className="text-sm text-gray-400 absolute right-4">
```

Comment:
This button doesn't do anything. I assume that eventually it will clear the search and filter input?

Reply:
Correct